### DISCUSSION TITLE
[Bug] Genshin Impact quit unexpectedly

### DISCUSSION THREAD

**Discussion by PixelCookie-zyf** [PlayCover/PlayCover#402] [General]
I'm using playcover 2.0.0 on 2021 M1 pro 16-inches and it always quits unexpectedly when I switch teams.

<img width="291" alt="CleanShot 2022-10-05 at 23 49 00@2x" src="https://user-images.githubusercontent.com/82641144/194104860-7d5f6232-8183-4e2f-807e-86fdad12fa5e.png">
es

<img width="1216" alt="CleanShot 2022-10-05 at 23 51 25@2x" src="https://user-images.githubusercontent.com/82641144/194105404-c0bb3ad6-06ee-4bab-95e3-82b8a3bf95c6.png">

**Comment by PixelCookie-zyf** [comment]
Mac Version:MacOS Montery 12.6

**Reply by Archangel0049** [reply]
Are you using an external display?

**Comment by PixelCookie-zyf** [comment]
And when  I download the language packs, it always shows the connection problems!

**Comment by NoirJ0e** [comment]
customize resolution to 4:3 ratio, my personal setting is 1366*1024, this will fix this problem. Tested on build 111 and 125

**Reply by Archangel0049** [reply]
Naaa, I'm now using 4k resolution with 16:10 ratio, but in the boil-in display, it works

**Comment by Archangel0049** [comment]
I had the same problem, there's no connection between resolution or resolution ratio. You should simply starting the game with in build-in display and it works! One hour without errors for now.
